### PR TITLE
Xsrf done

### PR DESCRIPTION
**Pull request by Benet5**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (gossie) [inline package.json:1]: This file can be deleted. I think you executed some `npm` commands in the root directory.

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,195 @@
+{
```
Comment (gossie) [inline package-lock.json:1]: This file can be deleted. I think you executed some `npm` commands in the root directory.

File: backend/src/main/java/Billprint/Auth/SecurityConfig.java [hunk 3]
```diff
@@ -31,9 +37,13 @@ protected void configure (AuthenticationManagerBuilder auth) throws Exception{
         auth.userDetailsService(userDetailsService);
     }
 
+    // entweder müssen die profile in der Main , der yml oder den application properties gesetzt werden.
```
Comment (gossie) [inline backend/src/main/java/Billprint/Auth/SecurityConfig.java:40]: You can also set the profile in the "Run Configuration" in the IDE by adding `-Dspring.profiles.active=<profile-name>` as Program argument.

File: package.json (hunk 1 above)
Comment (Benet5) [reply package.json]: done

File: package-lock.json (hunk 2 above)
Comment (Benet5) [reply package-lock.json]: done

File: backend/src/main/java/Billprint/Auth/SecurityConfig.java (hunk 3 above)
Comment (Benet5) [reply backend/src/main/java/Billprint/Auth/SecurityConfig.java:40]: ok

**Review by gossie — APPROVED** [review]

**Pull request merged**